web2.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+   
+    <title>Joe Johnson portfolio</title>
+    <link rel="stylesheet" type="text/css" href="css/style2.css" />
+</head>
+
+<header> ABOUT ME PAGE | <a href="web.html">portfolio</a> </header>
+<body>
+
+    <h1>Joe Johnson</h1>
+    <h2>Experience</h2>
+<p>I recently graduated from a Front-End Web Development 
+    course at General Assembly where I learned HTML, CSS, JavaScript, 
+    jQuery and how to be an awesome Front-End Web Developer! During my
+     spare time, I enjoy senior water aerobics, pickling, and spending 
+     time with my goldfish, Jerry.
+
+    </p> 
+
+    <h3>portfolio</h3>
+
+    <p>
+            <ul>
+           <li>Web Developer, Relaxr 2015 - present
+            Developed a multi-column layout blog, landing page, 
+            and contact forms that render on mobile devices.</li>
+            
+            <img src="image/relaxr-deliverable.png" alt="Relaxr blog"> 
+            
+
+            <li>Web Developer, Startup Matchmaker 2015 - present
+            Used a design team's wireframes to develop this company's responsive homepage.
+            <img src="image/startup-matchmaker-deliverable.png" alt="Startup Matchmaker 2015"></li>
+
```

Comment:
To get the rest of the images to be centered like the first one you should put the < img > outside of the < li > tag. Try it and see what happens :)